userguide/sound_classifier/export-coreml.md

```diff
@@ -73,20 +73,19 @@ guard let audioData = try? MLMultiArray(shape:[windowSize as NSNumber], dataType
 }
 
 // Ignore any partial window at the end.
-var windowNumber = 0
 var results = [Dictionary<String, Double>]()
-while ((windowNumber+1) * windowSize) <= (wav_file.length - Int64(windowSize)) {
-   let offset = windowNumber * windowSize
+let windowNumber = wavFile.length / Int64(windowSize)
+for windowIndex in 0..<Int(windowNumber) {
+   let offset = windowIndex * windowSize
    for i in 0...windowSize {
-      audioData[i] = NSNumber.init(value: bufferData[0][offset + i])
+       audioData[i] = NSNumber.init(value: bufferData[0][offset + i])
    }
    let modelInput = modelInput(audio: audioData)
 
    guard let modelOutput = try? model.prediction(input: modelInput) else {
-      fatalError("Error calling predict")
+       fatalError("Error calling predict")
```

Comment:
Please also remove the added whitespace here too.

Reply:
Done.